composer.json

```diff
@@ -100,9 +100,9 @@
     ],
     "scripts": {
         "clone-addon": ".tools/bin/clone-addon",
-        "refresh": ".tools/bin/refresh",
+        "refresh": "bash .tools/bin/refresh",
```

Comment:
Hmm, ich nutze normalerweise eine zsh, wechselt es dann bei mir zur bash?

Reply:
hmm, wobei das eventuell sowieso passiert, da die dateien das als shebang enthalten. daher vielleicht auch egal

Reply:
> hmm, wobei das eventuell sowieso passiert, da die dateien das als shebang enthalten. daher vielleicht auch egal

ich vermute ja. mit 100% sicherheit kann ich es nicht sagen.

Reply:
ausserdem vermute ich dass in der zsh der "bash" befehl ggf. umleitet zu zsh selbst